llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -10101,6 +10101,13 @@ bool LoopVectorizePass::processLoop(Loop *L) {
   // Get user vectorization factor and interleave count.
   ElementCount UserVF = Hints.getWidth();
   unsigned UserIC = Hints.getInterleave();
+  if (LVL.hasUncountableEarlyExit() && UserIC != 1 &&
+      !VectorizerParams::isInterleaveForced()) {
+    UserIC = 1;
+    reportVectorizationInfo("Interleaving not currently supported for loops "
+                            "with uncountable early exits",
+                            "InterleaveEarlyExitDisabled", ORE, L);
```

Comment:
```suggestion
    reportVectorizationInfo("Interleaving not supported for loops "
                            "with uncountable early exits",
                            "InterleaveEarlyExitDisabled", ORE, L);
```
Not a strongly held view but I prefer to simply state the facts and also "currently" is implied by the fact the user has just tried it.